magprime/templates/regextra.html

```diff
@@ -10,6 +10,7 @@
         }
         $("input:radio[name='amount_extra'][value=40]").parents('.btn').append($('#guitar_keychain'));
         $('#guitar_keychain').on('click', function(e) { e.stopPropagation(); });
+        {% if c.BEFORE_SUPPORTER_DEADLINE %}
         if ($("input:radio[name='amount_extra'][value={{ c.SUPPORTER_LEVEL }}]")) {
```

Comment:
Ah, I think I see why this was even necessary in the first place.  This Javascript if-statement will always return a truthy value, because all jQuery collections are truthy.  In other words:

```javascript
if ($('#id-which-does-not-exist')) {
    // we WILL get here, because the above is truthy
}
```

The correct way to check for whether something exists on the page is with ``.size()``, e.g.

```javascript
if ($("input:radio[name='amount_extra'][value={{ c.SUPPORTER_LEVEL }}]").size()) {
    ...
}
```

If memory serves, I was super slammed with something or other (probably MIVS stuff) when this was originally implemented, so I have less understanding of this code than I usually do, but at a glance it looks like this Javascript if statement is trying to accomplish the same thing as your Django template if statement.  If that's true, we could probably just fix the Javascript if statement to be correct and then call it a day.  Alternatively, we could keep the Django if statement and then just remove the Javascript if statement, since it's pointless.